README.md

```diff
@@ -1,17 +1,19 @@
 # Vorpal
 
-
-[![Build Status](https://travis-ci.org/dthree/vorpal.svg)](https://travis-ci.org/dthree/vorpal/)
-<a href="https://www.npmjs.com/package/vorpal">
-  <img src="https://img.shields.io/npm/dt/vorpal.svg" alt="NPM Downloads" />
-</a>
+[![NPM Version](https://img.shields.io/npm/v/vorpal.svg)](https://www.npmjs.com/package/vorpal)
+[![Status](https://img.shields.io/badge/status-Refactor%20in%20progress-yellow.svg)](https://github.com/vorpaljs-reforged/vorpal/projects/1)
+[![Build Status](https://travis-ci.com/vorpaljs-reforged/vorpal.svg)](https://travis-ci.com/vorpaljs-reforged/vorpal/)
+[![NPM Downloads](https://img.shields.io/npm/dt/vorpal.svg)](https://www.npmjs.com/package/vorpal)
 [![Package Quality](http://npm.packagequality.com/shield/vorpal.svg)](http://packagequality.com/#?package=vorpal)
-<a href="https://www.npmjs.com/package/vorpal">
-  <img src="https://img.shields.io/npm/v/vorpal.svg" alt="NPM Version" />
-</a>
 [![XO code style](https://img.shields.io/badge/code_style-XO-5ed9c7.svg)](https://github.com/sindresorhus/xo)

```

Comment:
Are we sticking with Xo or dropping it in favor of prettier or standardjs?

Could changing our code styles could prevent us from getting merge approval into the original repo? Again, not a blocker on this PR but if we want to swap that out it wouldn’t be a bad time to change it here

Reply:
I don't know.
Would like @vorpaljs-reforged/vorpaldevelopers inputs.
But getting ride of the XO badge can be done in a second time

Reply:
xo can and should be used in addition of prettier. We could replace xo with a different linter, but xo and prettier are mutually exclusive

Reply:
I have no experience with xo yet, so no strong opinion on this. I'd say leave it in for now.

Reply:
Yeah I have absolutely no experience with xo so I didn’t know if we had to pick one over the other. Leave it for now!